features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1793,6 +1793,7 @@
         {% if authentication.jit_provisioning.allow_idp_login is defined %}
             <AllowLoginToIDP>{{authentication.jit_provisioning.allow_idp_login}}</AllowLoginToIDP>
         {% endif %}
+        <AllowAssociatingToExistingUser>{{authentication.jit_provisioning.associating_to_existing_user}}</AllowAssociatingToExistingUser>
```

Comment:
Lets add this config to identity.xml file as well

Reply:
Since this is a boolean value and need no examples, would adding this to the identity.xml file serve any purpose?

Reply:
We added this without having a if condition. So this will always be available in identity.xml file after server startup. So it is better if we can show this in default pack as well. 
Actually, there is no main usecase of adding this to identity.xml file other than visualization. It will be helpful for devs to see how this config would looks like. Because checking at identity.xml.j2 file, we cannot see what type of value this can be.  We know it is a boolean, but someone who is looking fresh might not.

Reply:
added to default.json file through https://github.com/wso2/carbon-identity-framework/pull/5468/commits/fb117f2cd78ccb6cad0eb90a069840bbe90ffdcb